src/index.ts

```diff
@@ -840,4 +840,65 @@ export default class Purchases {
   ): Promise<{ [productId: string]: IntroEligibility }> {
     return RNPurchases.checkTrialOrIntroductoryPriceEligibility(productIdentifiers);
   }
+
+
+  /**
+   * Invalidates the cache for purchaser information.
+   * This is useful for cases where purchaser information might have been updated outside of the app, like if a
+   * promotional subscription is granted through the RevenueCat dashboard.
+   */
+  public static invalidatePurchaserInfoCache() {
+    RNPurchases.invalidatePurchaserInfoCache();
+  }
+
+  /**
+   * Subscriber attributes are useful for storing additional, structured information on a user.
+   * Since attributes are writable using a public key they should not be used for
+   * managing secure or sensitive information such as subscription status, coins, etc.
+   *
+   * Key names starting with "$" are reserved names used by RevenueCat. For a full list of key
+   * restrictions refer to our guide: https://docs.revenuecat.com/docs/subscriber-attributes
+   *
+   * @param attributes Map of attributes by key. Set the value as an empty string to delete an attribute.
+   */
+  public static setAttributes(attributes: { [key: string]: string }) {
+    RNPurchases.setAttributes(attributes);
+  }
+
+  /**
+   * Subscriber attribute associated with the email address for the user
+   *
+   * @param email Empty String or null will delete the subscriber attribute.
+   */
+  public static setEmail(email: string) {
```

Comment:
I would indicate that attributes can also be null as `string | null`

Reply:
awesome! didn't know you could do that. will do